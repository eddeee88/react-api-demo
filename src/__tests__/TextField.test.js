import React from "react";
import TextField from "./../common/components/TextField";
import renderer from "react-test-renderer";

it("TextField renders correctly", () => {
  const tree = renderer
    .create(
      <TextField
        id="TestId"
        className="TestName"
        placeholder="TestPlaceholder"
        label="TestLabel"
        value="TestValue"
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
