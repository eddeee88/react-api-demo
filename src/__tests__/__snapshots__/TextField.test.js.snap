// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField renders correctly 1`] = `
<div
  className="MuiFormControl-root-1 MuiFormControl-fullWidth-4 TestName"
>
  <label
    className="MuiFormLabel-root-10 MuiInputLabel-root-5 MuiInputLabel-formControl-6 MuiInputLabel-animated-9 MuiInputLabel-shrink-8"
    data-shrink={true}
    htmlFor="TestId"
  >
    TestLabel
  </label>
  <div
    className="MuiInput-root-15 MuiInput-fullWidth-22 MuiInput-formControl-16 MuiInput-underline-19"
  >
    <input
      aria-invalid={false}
      className="MuiInput-input-23"
      disabled={false}
      id="TestId"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="TestPlaceholder"
      type="text"
      value="TestValue"
    />
  </div>
</div>
`;
